docker/manage

```diff
@@ -189,6 +189,12 @@ case "$COMMAND" in
     configureEnvironment $@
     build ${_startupParams}
     ;;
+  test)
+    _startupParams=$(getStartupParams $@)
+    configureEnvironment "$@"
+	  build
```

Comment:
You may want to use `build ${_startupParams}` as in line `190` for consistency, unless there is a specific reason not to.

Reply:
This was committed accidentally. Will remove